lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/GroupCommClient.cs

```diff
@@ -80,7 +80,7 @@ public sealed class GroupCommClient : IGroupCommClient
                     group.WaitingForRegistration();
                 }
             }
-            catch (SystemException e)
+            catch (Exception e)
```

Comment:
PR still has `catch (Exception e)` here. Have you posted the latest version of your code?